### ISSUE TITLE
3rd Party Clients menu not an actual sub menu (just a menu entry without functionality)

### ISSUE THREAD

**Issue by Carighan** [JosefNemec/Playnite#2717]
### Bug Description

The menu entry '3rd Party Clients' seems to be non-functional since I believe the `9.12` update. I sadly cannot truly guarantee it happened since that update, I don't have screenshots showing the client before that.

What happens is that both in the main menu and in the context menu of the taskbar icon, the entry looks like this:
![image](https://user-images.githubusercontent.com/33469/146687941-c238f9e9-c5eb-47e6-b801-03bce188806b.png)
![image](https://user-images.githubusercontent.com/33469/146687962-e2d289b9-9e50-4295-ae00-667822d486c7.png)

The arrow indicating a submenu is missing, and in fact there is no submenu. I can highlight the entry, but nothing happens, and if I click on it he menu closes as if I had selected an option but, well, nothing happens.

### To Reproduce

1. Start Playnite
2. Open either the main menu or the taskbar context menu.
3. Try to interact with the 3rd Party Clients submenu.

Things I've tested:
* Starting in safe mode.
* Using a different theme than Stardust, or no theme at all.

Neither of these made a difference. I have the following libraries enabled: battle.net, Epic, GOG, Humble, Nintendo, Origin, Steam, Xbox. I have not tried changing this configuration yet.

### Diagnostics ID

272f6771-85ed-4274-840f-951a2329d225

**Comment by JosefNemec** [comment]
Please try with this build of [Origin plugin](https://github.com/JosefNemec/Playnite/files/7741451/OriginTest.zip). It for some reason fails to detect where Origin is installed and it blows up everything.

Unpack to Addons -> Installed -> Libraries -> Origin integration, there's `Installation folder` button.

**Comment by Carighan** [comment]
Yep, that did the trick! Should I close this ticket as that fix already exists?

**Comment by JosefNemec** [comment]
No I need to make changes in Playnite to prevent plugins from breaking this completely.
